core/src/test/java/org/apache/iceberg/DataTableScanTestBase.java

```diff
@@ -180,12 +184,25 @@ public void testSettingInvalidRefFails() {
 
   private void validateExpectedFileScanTasks(ScanT scan, List<CharSequence> expectedFileScanPaths)
       throws IOException {
+    validateExpectedFileScanTasks(scan, expectedFileScanPaths, null);
+  }
+
+  private void validateExpectedFileScanTasks(
+      ScanT scan,
+      Collection<CharSequence> expectedFileScanPaths,
+      Map<CharSequence, String> fileToManifest)
```

Comment:
I think this should be either `Map<String, String>` or `CharSequenceMap` to be safe.

Reply:
Good point, updated to use CharSequenceMap